crawl-ref/source/spl-book.cc

```diff
@@ -514,7 +514,9 @@ bool library_add_spells(vector<spell_type> spells)
     {
         vector<string> spellnames(new_spells.size());
         transform(new_spells.begin(), new_spells.end(), spellnames.begin(), spell_title);
-        mprf("You add the spell%s %s to your library.",
+        mprf(you.num_turns < 1
```

Comment:
While not re-adding the quiet bool is good, this should simply not print a message at game start. The only case where we want to announce the starting library is for Wanderers, which are already handled elsewhere.